### ISSUE TITLE
[Beta] Applied Accessibility - Tests fails to check for closing </audio> tag

### ISSUE THREAD

**Issue by Asjas** [freeCodeCamp/freeCodeCamp#16419]
#### Challenge Name

https://beta.freecodecamp.org/en/challenges/applied-accessibility/improve-accessibility-of-audio-content-with-the-audio-element

#### Issue Description

The issue with this challenge is that you can use the opening `<audio>` tag and leave out the closing `</audio>` tag and the test passes. The test should be checking if the closing tag is also used.

#### Browser Information

* Browser Name, Version: Chrome, Version 63.0.3239.108 (Official Build) (64-bit)
* Operating System: Windows 10 Pro
* Mobile, Desktop, or Tablet: Desktop

#### Your Code

```js
<body>
  <header>
    <h1>Real Coding Ninjas</h1>
  </header>
  <main>
    <p>A sound clip of Zersiaxs screen reader in action.</p>

    <audio id="meowClip" controls>
    <source src="https://s3.amazonaws.com/freecodecamp/screen-reader.mp3"  type="audio/mpeg" />

  </main>
</body>
```
#### Screenshot

![audio](https://user-images.githubusercontent.com/3828967/34638511-a24fe846-f2d5-11e7-8114-45fd8b0aef5b.jpg)

**Comment by mikeATL** [comment]
This is not working correctly.  It does not detect the </audio> closing tag and gives an error when it is there.
<img width="570" alt="screen shot 2018-02-14 at 1 14 28 pm" src="https://user-images.githubusercontent.com/36350834/36220571-105c0212-1189-11e8-8d6b-c7f9bda2df57.png">

**Comment by Gulshanrajput** [comment]
Still facing the issue, tried everything in this thread.

**Comment by AaronWands** [comment]
Facing the same problem. It says that there needs to be a closing audio tag but I have one already. Not sure what to do.

**Comment by bloo0178** [comment]
I also had the same problem. I removed the id entirely and the test passed.
`<audio controls> </audio>`
